shell/platform/windows/win32_window.cc

```diff
@@ -131,12 +132,28 @@ Win32Window::MessageHandler(HWND hwnd,
         break;
 
       case WM_MOUSEMOVE:
+        if (!tracking_mouse_events) {
+          TRACKMOUSEEVENT tme;
+          tme.cbSize = sizeof(tme);
+          tme.hwndTrack = hwnd;
+          // Not tracking Hover since the engine handles that logic.
+          tme.dwFlags = TME_LEAVE;
+          TrackMouseEvent(&tme);
+          tracking_mouse_events = true;
```

Comment:
How about tracking_mouse_leave since that's what it's actually being used for? The current name matches the API name, but is misleading otherwise (since we are getting most mouse events regardless of the state of this variable)

Reply:
Done